app/models/email_address.rb

```diff
@@ -49,6 +49,10 @@ def mil_email?
     email.end_with?('.mil')
   end
 
+  def self.last_sign_in
+    order('last_sign_in_at DESC NULLS LAST').first
+  end
```

Comment:
This looks like a scope? Can we mark it as one with
```suggestion
  scope :last_sign_in, -> { order('last_sign_in_at DESC NULLS LAST').first }
```

Reply:
In my opinion, a "scope" has the connotation of a filter and not an ordering. I followed the example described in the [Rails docs itself](https://api.rubyonrails.org/v8.0.1/classes/ActiveRecord/Scoping/Named/ClassMethods.html#method-i-scope) (see the last couple of sections).

Reply:
Also, scopes should return an `ActiveRecord::Relation` or `nil` whereas we are returning a single object.